test/squish/suite_VerifySettingsDialog/shared/scripts/gui_helpers.py

```diff
@@ -0,0 +1,166 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# -*- coding: utf-8 -*
+# mypy: disable-error-code="attr-defined"
+
+import gui_locators
+import squish
+import test
+
+snoozeTimeout = 3  # seconds
+
+
+def launchDeadlineConfigGUI():
+    squish.startApplication("deadline config gui")
+    test.log("Launching Deadline Config GUI")
+    test.log(
+        "Sleep for " + str(snoozeTimeout) + " seconds to allow GUI authentication to fully load."
+    )
+    squish.snooze(snoozeTimeout)
+    test.compare(
+        squish.waitForObjectExists(
+            gui_locators.aWS_Deadline_Cloud_workstation_configuration_DeadlineConfigDialog
+        ).visible,
+        True,
+        "Expect the Deadline Config GUI to be open.",
+    )
+
+
+def setGlobalSettings(profileName: str, jobHistDir: str):
+    test.compare(
+        str(squish.waitForObjectExists(gui_locators.profile_settings_QLineEdit).displayText),
+        jobHistDir,
+        "Expect job history directory default path to be equal.",
+    )
+    squish.mouseClick(
+        squish.waitForObjectItem(gui_locators.global_settings_AWS_profile_QComboBox, profileName),
+        484,
```

Comment:
If these are pixel placements for the mouse, I think we are going to want to do this another way. Doing it with mouse clicks makes our tests extremely fragile because any changes to the layout of the UI can potentially break our tests. 

Squish has ways of clicking on specific buttons directly which is more accurate and as long as we don't change the name of the underlying UI objects, our tests remain valid.

Check out this: https://doc.qt.io/squish/qt-convenience-api.html#qt-clickbutton-function